packages/fxa-payments-server/src/components/PaymentConfirmation/index.test.tsx

```diff
@@ -78,6 +102,55 @@ describe('PaymentConfirmation', () => {
     expect(footer).toBeVisible();
   });
 
+  describe('When payment_provider is "paypal"', () => {
+    const subject = () => {
+      return render(
+        <PaymentConfirmation
+          {...{
+            profile: userProfile,
+            selectedPlan,
+            customer: paypalCustomer,
+            productUrl,
+          }}
+        />
+      );
+    };
+
+    it('the paypal logo is visible', () => {
```

Comment:
nit: I think the convention is to start with a verb so that it reads `[the thing in describe][does this thing]`.  Not that we've been very consistent with it.